packages/data/src/registry.js

```diff
@@ -154,10 +159,15 @@ export function createRegistry( storeConfigs = {} ) {
 	};
 
 	//
-	// TODO:
-	// This function will be deprecated as soon as it is no longer internally referenced.
+	// Deprecated
 	//
 	function use( plugin, options ) {
+		deprecated( 'registry.use', {
+			alternative: 'registry.registerGenericStore',
+			plugin: 'Gutenberg',
+			version: '4.6.0',
```

Comment:
We should remove the version here, we can't remove APIs anymore. We need to figure out the deprecation strategy first.

Reply:
I'm not sure what the next steps are for this now that it can't be removed. So whatever you want to do is fine.